document/CoreFormat.mdk

```diff
@@ -1087,28 +1167,13 @@ that identifies the file where the static analysis tool produced the result.
 Whether `resultFile` is present depends on the information available to the tool that produces the log file
 (see &sect;[#location-Constraints]).
 
-### `logicalLocation` property { #location-logicalLocation }
```

Comment:
Most of this text moved up into `run.logicalLocations` (see above, in the right-hand diff).